content/docs/reference/pkg/_index.md

```diff
@@ -1,9 +1,9 @@
 ---
-title: API Reference
+title: API / SDK Reference
```

Comment:
I think we should keep this "API Reference"

Reply:
@cnunciato - thoughts? We were talking about the "API" label being misleading, but I'm sort of brain dead now from all the TOC changes. :smile:

Reply:
Ah! Yes. I do have a few thoughts to share for your consideration. 😄 

* It's a bit odd IMO that the JS/TS SDKs all come under the heading "Node.js", as Node is primarily a runtime, where Python is primarily a language. Since languages are our thing, it'd seem clearer to me to name the section  "JavaScript", "TypeScript", "JavaScript/TypeScript", or the like. (Until I became familiar with how our docs are organized, I'd scan right past Node.js looking for something else.) I do see this differs from Heroku, for example (which features "Node.js"), but that seems okay to me, since Heroku is all about running services, so as a developer, you go in looking for Node specifically.

* I remember talking about API vs. SDK, yeah. I believe I threw out referring to this section in some way as SDKs or Languages (as I tend to equate "API" with "REST API"), but I think API Reference is fine, provided the things under that heading are clearly languages. It's the Node.js thing that's always struck me as a bit off.

Reply:
Regarding JS/TS, I like the suggestion to focus on "language" more than "runtime." I also think over time we'll want specific guides fo JavaScript versus TypeScript, similar to how we show both differently throughout our docs for code samples, etc.